config/sys/playbooks/setup.yml

```diff
@@ -0,0 +1,144 @@
+---
+- name: "Provision EC2"
+  hosts: "web-1"
+  become: true
+
+  # VARS
+  # ############################################################################
+
+  vars:
+    domain: poser.pugx.org
+    email: pugx@example.com
```

Comment:
`example.com`? @garak we have an email address for the `pugx.org`?

Reply:
I'm afraid not 😔

Reply:
`pugroma at gmail.com`?

Reply:
That's an email we're using for PUG Roma twitter account.
If you think it's suitable here, feel free to use it

Reply:
done